src/components/duration.rs

```diff
@@ -447,68 +430,37 @@ impl Duration {
         }
     }
 
-    /// Rounds the current `Duration`.
     #[inline]
     pub fn round(
         &self,
-        increment: Option<RoundingIncrement>,
-        smallest_unit: Option<TemporalUnit>,
-        largest_unit: Option<TemporalUnit>,
-        rounding_mode: Option<TemporalRoundingMode>,
+        options: Option<RoundingOptions>,
```

Comment:
I would make this take the `RoundingOptions` directly, because in the case where someone wants to set an option, they would need to double-wrap with `Some`, which is not ideal.